hardhat.config.ts

```diff
@@ -44,6 +44,9 @@ function etherscan() {
 
 const config: HardhatUserConfig = {
   networks: {
+    hardhat: {
+      allowUnlimitedContractSize: true
```

Comment:
This was added to enable local testing with contracts larger than 24Kb. Not sure if we want to keep this around for future local testing, or get rid of it

Reply:
we should be able to remove this now